index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
                 <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
             </ul>
         </section>
+
+
+    </div>
+
+    <!-- card 1 -->
+    <div class="work-container">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-1.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Tonic</h1>
+
```

Comment:
- [ ] You have done great by ensuring that the heading uses `hx` HTML tag 👏 . However, in this section let's consider using `h2-h6`. This is because to be [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct, the page should have only one `h1` which is the main heading of the page. In addition, doing this increases the accessibility of the page 👍 .